src/tree/durableTaskScheduler/DurableTaskSchedulerErrorWorkspaceResourceModel.ts

```diff
@@ -0,0 +1,26 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { type ProviderResult, ThemeIcon, TreeItem } from "vscode";
+import { type DurableTaskSchedulerWorkspaceResourceModel } from "./DurableTaskSchedulerWorkspaceResourceModel";
+
+export class DurableTaskSchedulerErrorWorkspaceResourceModel implements DurableTaskSchedulerWorkspaceResourceModel {
+    constructor(
+        private readonly error: string) {
+    }
+
+    getChildren(): ProviderResult<DurableTaskSchedulerWorkspaceResourceModel[]> {
```

Comment:
Should this item have any children? I believe it'll make it expandable, but then nothing would show below which is an odd UI experience.

Reply:
The `TreeItemCollapsableState` controls whether the expansion UX is shown (and, by default, not).  That said, I'll update the model interface to make this optional so it can be omitted, and be clearer, for those that don't have children.

Reply:
I think you missed this one when removing `getChildren` definitions.

Reply:
Bah!  Sorry about that.